src/main/java/com/school/schedule/app/School.java

```diff
@@ -50,4 +66,14 @@ public void setTelephone( float telephone ) {
         this.telephone = telephone;
     }
 
+    public void setStudents( List students ) {
```

Comment:
Instead of setter methods for the students and teachers lists, it would be cleaner to create add methods which simply append a student or teacher to their respective lists.
Remove methods can be ignored for this app release.

Reply:
thank you for the review, fixed it in the new commit!